src/ng2-modal.ts

```diff
@@ -1,7 +1,7 @@
 import {Component, Input, Output, EventEmitter, ElementRef, ViewChild} from "@angular/core";
 
 @Component({
-    selector: "modal",
+    selector: "ng2-modal",
```

Comment:
I dont see any good reason to rename it right now. I would wait for upcoming webcomponents spec draft and see if others would follow it too. 

> The reason they claim is so there will be less conflicts between spec elements and user custom elements.
I think this is more recommendation rather then a practical reason. In practice updates come from those guys rarely once per few years and more. In such period we will see angular 3 and new version of modal maybe with different name if it conflicts